README.md

```diff
@@ -29,6 +29,51 @@ How Do I Get It?
 How Do I Use It?
 ---
 
+### In node (JavaScript):
```

Comment:
I think we should remove the CoffeeScript examples, and then this can simply be "In node".